influxql/query_executor.go

```diff
@@ -148,13 +151,13 @@ func (e *QueryExecutor) Close() error {
 }
 
 // ExecuteQuery executes each statement within a query.
-func (e *QueryExecutor) ExecuteQuery(query *Query, database string, chunkSize int, closing chan struct{}) <-chan *Result {
+func (e *QueryExecutor) ExecuteQuery(query *Query, database string, chunkSize int, readonly bool, closing chan struct{}) <-chan *Result {
```

Comment:
I agree. Most of this stuff is included in `ExecutionContext` so that can probably just be used. We're going to have to change this at some point to include the user, so I'll change it then.